src/PlaygroundGrains/BuildGrain.cs

```diff
@@ -0,0 +1,114 @@
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.IO.Compression;
+using System.Linq;
+using System.Threading.Tasks;
+using GrainInterfaces;
+
+namespace Grains;
+
+public class BuildGrain : ProcessGrain<BuildRequestDto, BuildResponseDto>, IBuildGrain
```

Comment:
We can kill this grain immediately and do clean up of storage by doing `DeactivateOnIdle();`

Then we do clean up in `OnDeactivateAsync`

This way we make sure we clean up once we are done.